### DISCUSSION TITLE
How to add bias in linear and convolutional layers?

### DISCUSSION THREAD

**Discussion by CloudyDory** [lava-nc/lava-dl#175] [Q&A] [answered]
Hi, it seems that the linear and convolutional layer in sLayer module disable the bias term. If I have an SNN network with bias term, how can I run the network under LAVA?

**Comment by bamsumit** [comment, marked as answer]
Hi @CloudyDory, bias is a property of neurons in neuromorphic processors. That is why they are disabled on the synapses. It is not fully exposed to the users for neurons as well. They are triggered only when one uses normalization on neurons.

It seems that you want to run inference on your existing network, not train it. Is that correct? In that case, you can initialize the bias of the neuron process in lava. If you are using NetX specification, you can also add `'bias'` property of a layer.
> Note: bias for LIF neuron gets added to the voltage state, not current.
